stub_api_server/get_mesh.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
```

Comment:
Any reason to not write this in Python? It may allow to even create mesh responses with desired density of data. BTW, how do you create the 40% and 80% mesh results today?

Reply:
Hmm didn't even consider employing python here, shell script does just the job.
Paused the test for a few minutes, then resumed and fetched results for subsequent time windows of 5 minutes width, this got me 40%, 80%, 100% complete data